apps/sv/frontend/src/__tests__/governance/governance-create-unallocated-unclaimed-activity-record.test.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { describe, test, expect } from 'vitest';
+import { MemoryRouter } from 'react-router-dom';
+import { SvConfigProvider } from '../../utils';
+import dayjs from 'dayjs';
+import CreateUnallocatedUnclaimedActivityRecord from '../../components/votes/actions/CreateUnallocatedUnclaimedActivityRecord';
+
+describe('UnallocatedUnclaimedActivityRecord Amount Validation', () => {
```

Comment:
@fayi-da please make sure we still support this in the new governance UI you are working on